gaphor/UML/classes/classespropertypages.py

```diff
@@ -77,18 +77,30 @@ def construct(self):
 
         builder = new_builder(
             "classifier-editor",
-            signals={"abstract-changed": (self._on_abstract_change,)},
+            signals={
+                "abstract-changed": (self._on_abstract_change,),
+                "isFinalSpecialization-changed": (
+                    self._on_isFinalSpecialization_change,
+                ),
+            },
         )
 
         abstract = builder.get_object("abstract")
         abstract.set_active(self.subject.isAbstract)
 
+        ifFinalSpecialization = builder.get_object("isFinalSpecialization")
```

Comment:
PEP 8 conventions for Python have variables lowercase with underscores separating names. Even if the UML model calls it that, I don't think we want it written like that in the code.

Reply:
I changed all (I think) the capitalization to conform with the policies you suggested.